sdk/maps/azure-maps-render/src/main/java/com/azure/maps/render/RenderClient.java

```diff
@@ -158,8 +181,14 @@ public MapAttribution getMapAttribution(TilesetID tilesetId, int zoom, List<Doub
      * @return the response.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public InputStream getMapStateTile(String statesetId, TileIndex tileIndex) {
-        return this.serviceClient.getMapStateTile(statesetId, tileIndex);
+    public byte[] getMapStateTile(String statesetId, TileIndex tileIndex) {
```

Comment:
This is inconsistent with getMapTile(). As we discussed, all should return InputStream fetched from StreamResponse.